tests/jballerina-debugger-integration-test/src/main/java/org/ballerinalang/debugger/test/utils/DebugTestRunner.java

```diff
@@ -415,6 +416,29 @@ public void resumeProgram(StoppedEventArguments context, DebugResumeKind kind)
         return stackFrames;
     }
 
+    /**
+     * Can be used to fetch threads when a debug hit is occurred.
+     *
+     * @return Thread array with threads information.
+     * @throws BallerinaTestException if an error occurs when fetching threads.
+     */
+    public org.eclipse.lsp4j.debug.Thread[] fetchThreads() throws BallerinaTestException {
```

Comment:
Shall we avoid using full qualified names?

Reply:
I'm using the full qualified name to avoid conflicts between `java.lang.Thread[]` and `org.eclipse.lsp4j.debug.Thread[]`.  Here Thread[] already referring to `java.lang.Thread[]`. Therefore I'm using full qualified name `org.eclipse.lsp4j.debug.Thread[]` to refer debug Thread[].

Reply:
Yeah I also noticed that this is unavoidable. +1